instrumentation/elastic-opentelemetry-instrumentation-openai/src/opentelemetry/instrumentation/openai/helpers.py

```diff
@@ -292,48 +290,61 @@ def _send_log_events_from_messages(
         if message["role"] == "system" or message["role"] == "developer":
             if message["role"] == "developer":
                 body["role"] = message["role"]
-            event = Event(name=EVENT_GEN_AI_SYSTEM_MESSAGE, body=body, attributes=attributes)
-            event_logger.emit(event)
+            # keep compat on the exported attributes with Event
+            log = LogRecord(
+                event_name=EVENT_GEN_AI_SYSTEM_MESSAGE,
+                body=body,
+                attributes={**attributes, "event.name": EVENT_GEN_AI_SYSTEM_MESSAGE},
+            )
+            logger.emit(log)
```

Comment:
I'm disappointed that this is the API. Keyword arguments are pythonic and classes less so, so I wonder are other otel-python maintainers actually familiar with python?

Reply:
The other PR fixed being able to work with the API LogRecord, this one that I just merged added the possibility to use the separated parameters instead of passing a LogRecord: https://github.com/open-telemetry/opentelemetry-python/pull/4737